e2e-cypress/files/support/test-evidence.ts

```diff
@@ -17,7 +17,7 @@ const logEvidence = (name: string, step: number, description: string, evidenceLo
   });
 }
 
-export const printTestDOMEvidence = (testName: string, testStep: number, selector: string, description: string) => {
+export const printTestDOMEvidence = (testName = 'unknown', testStep: number, selector: string, description: string) => {
```

Comment:
TBH I would not introduce default values at this spot. It has the following disadvantages:
* `'unknown'` is a more or less opinionated term that would now be active for all projects using this.
* The input `testName` now has type `string | undefined`. This allows accidentially handing in a potentially undefined value. Using a default value at this spot would make it even harder to spot the mistake.
* This function and also the others have the purpose to provide per-test evidence. Offering the possibility to miss out `testName` doesn't make sense for this purpose.

I would leave `testName` without default setting, but use `'unknown'` when handing in the test name in `acceptance.spec.cy.ts`. There I would also make use of a const.